package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@warp-ds/fonts",
   "type": "module",
-  "version": "1.1.0",
+  "version": "1.1.1-next.1",
```

Comment:
Hmm should this be 1.1.1?

Reply:
This will be updated by `semantic-release` once a new release is cut and `semantic-release-bot` pushes a commit to `main`. Like it did [here](https://github.com/warp-ds/fonts/commit/cc6da402fa28081adc6e86e9a25773ecbf49a1e4).